Proposals/xgov-33.md

```diff
@@ -0,0 +1,57 @@
+---
+title: FairTransfer
+author: 1m1 (1m1-github)
+company_name: 2i2i
+category: dApps
+focus_area: Defi
+open_source: Yes
+amount_requested: 400000
+status: Draft
```

Comment:
Remember to change this to "Final" if you want your proposal to go through the next voting session.